megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -826,6 +826,20 @@ private JMenuItem createMenuItem(JMenu m, String label, String command) {
      * @param event - the <code>ActionEvent</code> that spawned this call.
      */
     public void actionPerformed(ActionEvent event) {
+        
+        // Changes that are independent of the current state of MM
+        if (event.getActionCommand().equals(ClientGUI.VIEW_INCGUISCALE)) {
+            float guiScale = GUIPreferences.getInstance().getGUIScale();
+            if (guiScale < ClientGUI.MAX_GUISCALE) {
+                GUIPreferences.getInstance().setValue(GUIPreferences.GUI_SCALE, guiScale + 0.1);
+            }
+        } else if (event.getActionCommand().equals(ClientGUI.VIEW_DECGUISCALE)) {
+            float guiScale = GUIPreferences.getInstance().getGUIScale();
+            if (guiScale > ClientGUI.MIN_GUISCALE) {
+                GUIPreferences.getInstance().setValue(GUIPreferences.GUI_SCALE, guiScale - 0.1);
+            }
+        }
```

Comment:
Should we return at the end of these two, even if the if statement doesn't occur?

Reply:
You confuse me :) I thought about returning when the action command is in fact one of the GUISCALE things. But I thought well why not pass it on, if for whatever reason the other components want to do something with it as well. So, no return there, the event is always passed on to e.g. the ClientGUI, even if its a GUISCALE event.

Reply:
I missed the word internal, my apologies... but that makes a lot of sense :)